rules/align.smk

```diff
@@ -69,13 +69,13 @@ rule hisat2_align_bam:
 
 rule hisat2_merge_bams:
     input:
-        bams=expand("data/{sra}.sorted.bam", sra=config["sra"]) if check_sra() is True else expand("data/{fq1}.sorted.bam", fq1=config["fq1"])
+        bams=expand("{{dir}}/{sra}.sorted.bam", sra=config["sra"]) if check_sra() is True else expand("{{dir}}/{fq1}.sorted.bam", fq1=config["fq1"])
```

Comment:
What do the double brackets do?

Reply:
it prevents from expanding the `dir` wildcard while expanding the `sra` wildcard

Reply:
without the double brackets, `dir` undefined error would be thrown because it's attempting to expand `dir`

Reply:
Cool!